blueprints/addon/index.js

```diff
@@ -166,7 +166,12 @@ module.exports = {
     let addonFilesPath = this.filesPath(this.options);
     let addonFiles = walkSync(addonFilesPath);
 
-    return uniq(appFiles.concat(addonFiles));
+    let files = [].concat(
+      appFiles.filter((f) => f !== 'ember-cli-build.js'),
+      addonFiles
+    );
```

Comment:
Hmm, why do you need this? I would have expected that the `fileMap` just below here would have been updated to add:

```
'^ember-cli-build.js': 'tests/dummy/ember-cli-build.js'
```